dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -64,42 +64,38 @@ private constructor(
          * @param clientToken your API key of type Client Token
          */
         constructor(clientToken: String) :
-            this(clientToken, UUID(0, 0))
+                this(clientToken, UUID(0, 0))
 
         /**
          * A Builder class for a [DatadogConfig].
          * @param clientToken your API key of type Client Token
          * @param applicationId your applicationId for RUM events
          */
         constructor(clientToken: String, applicationId: String) :
-            this(clientToken, UUID.fromString(applicationId))
+                this(clientToken, UUID.fromString(applicationId))
 
         private var logsConfig: FeatureConfig = FeatureConfig(
             clientToken,
             applicationId,
             DatadogEndpoint.LOGS_US,
-            DEFAULT_SERVICE_NAME,
             DEFAULT_ENV_NAME
         )
         private var tracesConfig: FeatureConfig = FeatureConfig(
             clientToken,
             applicationId,
             DatadogEndpoint.TRACES_US,
-            DEFAULT_SERVICE_NAME,
             DEFAULT_ENV_NAME
         )
```

Comment:
you can set the default value here, that way you don't have to switch the order.